src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.jsx

```diff
@@ -302,22 +310,25 @@ const StyledButton = styled(Button)`
   color: ${p => p.theme.gray2};
 `;
 
+const color = {
+  Installed: 'success',
+  'Not Installed': 'gray2',
+  Pending: 'yellowOrange',
```

Comment:
Do the non values that aren't `Not Installed` get translated anywhere presently? Adding translations may not be a good idea if we're also using those values to do a lookup here.

Reply:
Maybe I just do the translation for the status [here](https://github.com/getsentry/sentry/pull/14541/files#diff-7ccd44fd59b2e3c616c16e439475167cR326)

Reply:
That would work.